最新版本1.0.10 cn.js 引入bug

Module not found: Error: Package path ./cn is not exported from package

> Module not found: Error: Package path ./cn is not exported from package

我遇到了一样的问题，

Package subbath './cn' is not defined by "exports" in  /node_modules/nzh/package.json

是 Node 应用中引入的，直接导致runtime报错应用启动不了，我先锁 1.0.9 版本解决了

--- a/package.json
+++ b/package.json
@@ -7,13 +7,19 @@
   "types": "nzh.d.ts",
   "exports": {
     ".": {
-      "types": "./nzh.d.ts"
+      "types": "./nzh.d.ts",
+      "import": "./nzh.js",
+      "require": "./nzh.js"
     },
     "./cn": {
-      "types": "./nzh.d.ts"
+      "types": "./nzh.d.ts",
+      "import": "./cn.js",
+      "require": "./cn.js"
     },
     "./hk": {
-      "types": "./nzh.d.ts"
+      "types": "./nzh.d.ts",
+      "import": "./hk.js",
+      "require": "./hk.js"
     }
   },
   "files": [

